workflow/hosts/gaea.yaml

```diff
@@ -0,0 +1,25 @@
+BASE_GIT: '/gpfs/f5/epic/proj-shared/global/glopara/data/git'
+DMPDIR: '/gpfs/f5/epic/proj-shared/global/glopara/data/dump'
+BASE_CPLIC: '/gpfs/f5/epic/proj-shared/global/glopara/data/ICSDIR/prototype_ICs'
+PACKAGEROOT: '/gpfs/f5/epic/proj-shared/global/glopara/data/nwpara'
+COMROOT: '/gpfs/f5/epic/proj-shared/global/glopara/data/com'
+COMINsyn: '${COMROOT}/gfs/prod/syndat'
+HOMEDIR: '/gpfs/f5/epic/scratch/${USER}'
+STMP: '/gpfs/f5/epic/scratch/${USER}'
+PTMP: '/gpfs/f5/epic/scratch/${USER}'
+NOSCRUB: $HOMEDIR
+ACCOUNT: epic 
+SCHEDULER: slurm
+QUEUE: batch
+QUEUE_SERVICE: batch
+PARTITION_BATCH: cron_c5 
+PARTITION_SERVICE: service
+CHGRP_RSTPROD: 'YES'
+CHGRP_CMD: 'chgrp rstprod'
+HPSSARCH: 'YES'
+HPSS_PROJECT: emc-global
+LOCALARCH: 'NO'
+ATARDIR: '${NOSCRUB}/archive_rotdir/${PSLOT}' 
```

Comment:
Does Gaea not have HPSS connectivity?
If so, this path should be pointing to HPSS location.